sections/healthcare_application/main.tex

```diff
@@ -2,4 +2,240 @@ \section{EMS - ED application}
 
 \subsection{Application}
 
-\subsection{Data Analysis}
\ No newline at end of file
+The system defined in section \ref{sec:model_overview} can be directly applied 
+to a healthcare scenario with two such queueing systems.
+The two queueing systems can be thought of as two Emergency Departments (ED) and 
+the Emergency Medical Services (EMS) as a distributor that distributes 
+patients to them.
+The parameters of the model can also be mapped into parameters of the ED and 
+the EMS.
+
+\begin{itemize}
+    \item \( \lambda_2 \): The rate of non-emergency patients that the EMS 
+    receives
+    \item \( \lambda_{1_i} \): The arrival rate of other patients to ED \(i\)
+    \item \( \mu_i \): The service rate of patients at ED \(i\)
+    \item \( C_i \): The number of available resources (beds) in ED \(i\)  
+    \item \( T_i \): The strategy that ED \(i\) chooses to play
+    \item \( N_i \): The total patient capacity of ED \(i\)
+    \item \( M_i \): The total parking capacity of ED \(i\)
+    \item \( R \): The time target for both EDs
+    \item \( \alpha \in [0, 1] \) : \textit{Importance} of blocking time and 
+    lost individuals (equation \ref{eq:obj-distributor})
+\end{itemize}
+
+The patients that are distributed by the EMS arrive at the hospital via an 
+ambulance and are then either unloaded at the ED or stay blocked outside in 
+the ambulance.
+Whether or not the ambulance and the patient stay blocked is determined by 
+the threshold that the given ED chooses to play.
+High threshold indicates that the ED accepts patients even if it is relatively 
+full, while low threshold means that the ED blocks ambulances more frequently.
+
+The game to be played now is between the two EDs and the EMS. 
+The strategies of the two EDs are the range of thresholds that they can choose
+from and their utilities is the proportion of individuals whose time in the 
+system is within a predetermined target time.
+The EMS has to decide how to distribute its patients among the two EDs so that 
+the blocking time of ambulances is minimised. 
```

Comment:
Yep, I see what you are saying. BUT, `alpha` is used for the EMS decision (i.e. what percentage to send to each hospital). So I think I should include something like that:

```suggestion
from and their utilities is the proportion of individuals whose time in the 
system is within a predetermined target time.
The EMS has to decide how to distribute its patients among the two EDs so that 
the weighted combination of the ambulance blocking time and the percentage of 
lost ambulances is minimised. 
```